src/rpc.ts

```diff
@@ -76,7 +74,7 @@ export class WatchClient {
    * last compaction revision.
    */
   public watch(): Promise<IDuplexStream<IWatchRequest, IWatchResponse>> {
-    return this.client.getConnection('Watch').then(cnx => cnx.watch());
+    return this.client.getConnection('Watch').then(cnx => (<any> cnx.client).watch());
```

Comment:
This is a limitation of TypeScript+protobuf unfortunately. It's a [client](https://github.com/mixer/etcd3/blob/password-auth/src/types/grpc.d.ts#L32) class which has methods beyond what we provide by default which are not known at compile time.